std/node/global.ts

```diff
@@ -1 +1,2 @@
+// @ts-ignore
 window["global"] = window;
```

Comment:
```suggestion
globalThis["global"] = globalThis;
```

Not your problem, but I didn't realise this was there, also I suspect we need in here:

```ts
global {
  var global: typeof globalThis;
}
```

So that it can be augmented properly without needing to ignore.